shell/platform/fuchsia/flutter/engine.cc

```diff
@@ -138,10 +134,10 @@ Engine::Engine(Delegate& delegate,
        view_token = std::move(view_token),
        view_ref_pair = std::move(view_ref_pair),
        max_frames_in_flight = product_config.get_max_frames_in_flight(),
-       vsync_handle = vsync_event_.get(), &view_embedder_latch]() mutable {
+       &view_embedder_latch]() mutable {
         session_connection_.emplace(
```

Comment:
At this point, this should be a shared_ptr too

Reply:
Done